scripts/e2e_setup_cluster.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+
+here="$(dirname "$(readlink --canonicalize "${BASH_SOURCE[0]}")")"
+root="$(readlink --canonicalize "$here/..")"
+RETRY_MAX=10
+INTERVAL=10
+TIMEOUT=300
+MULTUS_DAEMONSET_URL="https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/images/multus-daemonset.yml"
```

Comment:
URL is returing for 404, multus-daemonset.yml is deployments folder.
`MULTUS_DAEMONSET_URL="https://raw.githubusercontent.com/k8snetworkplumbingwg/multus-cni/master/deployments/multus-daemonset.yml"`